content/en/docs/concepts/containers/images.md

```diff
@@ -367,7 +367,31 @@ you must ensure all nodes in the cluster have the same pre-pulled images.
 This can be used to preload certain images for speed or as an alternative to authenticating to a
 private registry.
 
-All pods will have read access to any pre-pulled images.
+All pods will have read access to any pre-pulled images, unless the 
+[KubeletEnsureSecretPulledImages](#kubeletensuresecretpulledimages) feature gate
+is enabled.
+
+#### Pre-pulled image authentication {#prepulledimageauthentication}
+
+{{< feature-state feature_gate_name="KubeletEnsureSecretPulledImages" >}}
+
+If the `KubeletEnsureSecretPulledImages` 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+is enabled, Kubernetes will require valid authentication credentials for every pod
+image, even if that image is already present on the node. This is controlled by the
```

Comment:
That's not very accurate, the feature is much more complicated than that, we need to be more specific here.